components/compliance-service/reporting/relaxting/reports.go

```diff
@@ -126,36 +126,6 @@ func (backend ES2Backend) getNodeReportIdsFromTimeseries(esIndex string,
 		}
 	}
 
-	//When filtering by controls, we are reducing the array of report ids based on
```

Comment:
I removed this block as it's no longer needed. We are using only detailed indices with all filters being handled by `getFiltersQuery`